sdk/resourcemanager/Azure.ResourceManager.Core/tests/Scenario/HttpPipelineTests.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Azure.Core.TestFramework;
+using Azure.Identity;
+using NUnit.Framework;
+
+namespace Azure.ResourceManager.Core.Tests.Scenario
+{
+    [Parallelizable]
+    public class HttpPipelineTests : ResourceManagerTestBase
+    {
+        private ArmClient _client;
+
+        public HttpPipelineTests(bool isAsync)
+            : base(isAsync) //, RecordedTestMode.Record)
+        {
+        }
+
+        [SetUp]
+        public void SetUp()
+        {
+            _client = GetArmClient();
+        }
+
+        [Test]
+        public async Task ValidateHttpPipelines()
+        {
+            await _client.DefaultSubscription
+                .GetResourceGroups().Construct("westus")
+                .CreateOrUpdateAsync("test-CacheHttpPipeline");
```

Comment:
For asset names in order to run in parallel well use the create asset method provided

Recording.GenerateAssetName([prefix])

prefix can be whatever you want but randomness will be added to the end in order to make sure future tests never collide and create an asset with the same name